examples/basic/annotations/legends_item_visibility.py

```diff
@@ -1,3 +1,11 @@
+''' Marker and line plots that demonstrate manual control of legend visibility of individual items
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.LegendItem, bokeh.plotting.figure.circle, bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_annotations` > :ref:`ug_basic_annotations_legends_item_visibility`
```

Comment:
Last comment: we stopped doing the "breadcrumb" links with `>` because they are just too much of a burden to maintain  manually. These just need the direct reference to the section, which is simpler:
```suggestion
    :refs: :ref:`ug_basic_annotations_legends_item_visibility`
```